skyportal/models/phot_stat.py

```diff
@@ -333,9 +335,13 @@ def add_photometry_point(self, phot):
 
         filt = phot['filter']
         mjd = phot['mjd']
-        mag = -2.5 * np.log10(phot['flux']) + PHOT_ZP
+        if phot['flux'] > 0:
+            mag = -2.5 * np.log10(phot['flux']) + PHOT_ZP
+        else:
+            mag = np.nan
         snr = phot['flux'] / phot['fluxerr']
-        det = not np.isnan(snr) and snr > PHOT_DETECTION_THRESHOLD
+        det = phot['flux'] and phot['fluxerr']  # legal, non zero values
```

Comment:
Is this better than doing all on one line?

Reply:
I just didn't want the linter to fold it up in some bizarre way.